include/odp/api/spec/ipsec.h

```diff
@@ -1148,14 +1157,29 @@ typedef struct odp_ipsec_in_param_t {
  * Outbound inline IPSEC operation parameters
  */
 typedef struct odp_ipsec_out_inline_param_t {
-	/** Packet output interface for inline output operation
+	/** Packet output interface for inline outbound operation
 	 *
 	 *  Outbound inline IPSEC operation uses this packet IO interface to
 	 *  output the packet after a successful IPSEC transformation. The pktio
 	 *  must have been configured to operate in inline IPSEC mode.
 	 */
 	odp_pktio_t pktio;
 
+	/** Packet output TM queue for inline outbound operation
+	 *
+	 *  If pktio is configured to use traffic manager (ODP_PKTOUT_MODE_TM),
+	 *  then application is required to provide 'tm_queue' to be used after
+	 *  successful IPSEC transformation. The pktio must have been configured to
+	 *  operate in inline IPSEC mode.
```

Comment:
It could be set that this parameter is ignored if the pktio is not configured to use TM.

The comment about the need to enable inline IPsec mode in the pktio does not seem to belong here as it is a general requirement to inline IPsec.

Reply:
I'll reword this section to.

/** Packet output TM queue for inline outbound operation
 *
 *  Packets are send out through traffic manager if the packet output
 *  mode of the pktio is ODP_PKTOUT_MODE_TM. Such pktio interface can
 *  be used with inline IPSEC only if the platform has inline_ipsec_tm
 *  capability.
 *
 *  Application must set pktio as ODP_PKTIO_INVALID to indicate that
 *  the implementation should use tm_queue. This parameter would be
 *  ignored if the pktio is not configured for TM.
 *
 *  @see odp_ipsec_capability()
 */
odp_tm_queue_t tm_queue

Can you confirm is this looks fine?